src/main/webapp/i18n/de/assessment.json

```
@@ -28,7 +28,8 @@
             "button": {
                 "overrideAssessment": "Bewertung überschreiben",
                 "nextSubmission": "Nächste Abgabe bewerten",
-                "resolveConflict": "Konflikte behandeln"
+                "resolveConflict": "Konflikte behandeln",
+                "importExampleSubmission": "Import als Beispielabgabe"
```

```suggestion
                "importExampleSubmission": "Als Beispielabgabe importieren"
```
to keep in like with the "object verb" structure of the other strings